Performance control of embedded content/explainer.md

```diff
@@ -0,0 +1,157 @@
+The host app serves its main document with Document Policy directives to enforce on embedded content:<br>
+```
+Require-Document-Policy: basic, early-script, globals, script
+```
+
+Alternatively, the app can set any policy subset to individual frames:<br>
+```
+<iframe policy=”basic”>
+<iframe policy=”basic, early-script”>
+```
+
+As a result, requests to embedded content will be sent with `Sec-Required-Document-Policy` header matching the top-level document’s requested configuration. Per Document Policy design, embeddees receiving this request header must opt in for their content to be loaded, with the option to specify a reporting endpoint:
+
+```
+Document-Policy: basic, early-script, globals, script, *; report-to=endpoint
+Reporting-Endpoints: endpoint="https://example.com/reports"
+```
+
+### Discussion
+#### Using multiple Document Policy configuration points
+Document Policy allows for a single value for each configuration point. Since the categories proposed by this solution are independent of each other, this necessitates multiple configuration points.
+
+#### Opt-in and policy negotiation
+Document Policy requires that each document opts in to the policies to be applied on such document. While this is a limitation to amount of control the embedder can have over the embeddee's performance, we consider this behavior from Document Policy as necessary due to the following reasons:
+
+* **Control flow alteration.** Direct enforcement would allow for a document to impose changes to the control flow of unwitting embedded third parties.
+* **Potential for side-channel attacks.** See more details in Security and Privacy Considerations.
+
+Despite this limitation, Document Policy allows policy negotation which would allow the embedder to require the relevant policies in this proposal. Through this mechanism, the embedder can still require conformance from the embedee to be loaded, while keeping the embeddee in charge of its own control flow.
+
+#### Negotiation vs enforcement
+
+Document Policy proposes a mechanism for policy negotiation. An embeddee which doesn’t agree to the embedder’s policies will not be loaded. This document makes a distinction between this _negotiation_ (which might result in an embedee failing to load), and _enforcement_. Enforcement of the policy (what will happen when a violation occurs) is to be defined by each aspect of the proposed configuration points.
+
+#### Per-document constraints
+Document Policy directives are effective on a per-document basis. This generally makes it harder for constraints to have the desired impact to improve performance. In this document, we propose capping the overall complexity and resources through frame depth and count restrictions.
+
+**Open question:** what would be the appropriate frame count and depth limit? See related discussion in Security and Privacy Considerations.
```

Comment:
Usually we have an open question section at the end of explainers. Should we create a section for that and move all of those there?

Reply:
Moved these int a "open issues" section at the end.